docs/development-tips.md

```diff
@@ -105,4 +105,7 @@ by [mkdocs](https://www.mkdocs.org/user-guide/deploying-your-docs/).
 
 Every time a PR is merged to master, Travis will build and deploy the documentation to https://cfpb.github.io/cfgov-refresh. See [How we use Travis CI](https://github.com/cfpb/cfgov-refresh/blob/master/docs/travis.md) for more info.
 
-To add new pages to the navigation, edit the [mkdocs.yml](https://github.com/cfpb/cfgov-refresh/blob/master/mkdocs.yml) file in the root directory.
\ No newline at end of file
+To add new pages to the navigation, edit the [mkdocs.yml](https://github.com/cfpb/cfgov-refresh/blob/master/mkdocs.yml) file in the root directory.
+
+When running cfgov-refresh using [Docker-compose](https://cfpb.github.io/cfgov-refresh/installation/#docker-compose-installation)
+this documentation is running by default at http://localhost:8888.
```

Comment:
It's probably worth expanding this to include how to run it with a standard virtualenv setup, too.

Reply:
What, with `docker-compose up`? 😛 

Good idea. When we redo the IA of these docs it'd be nice to move the documentation meta-documentation to somewhere better.

Reply:
@Scotchester I've added a brief bit on running the docs without Docker, but I think I'm going to break out this section of the docs into their own "Contributing to the docs" document.